apps/drain-safe/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "drain-safe",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@gnosis.pm/safe-apps-react-sdk": "2.2.0",
+    "@gnosis.pm/safe-react-components": "^0.5.0",
+    "@material-ui/core": "^4.11.0",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.2.2",
+    "@testing-library/user-event": "^12.1.3",
+    "@types/jest": "^26.0.10",
+    "@types/node": "^14.6.2",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@types/styled-components": "^5.1.2",
```

Comment:
Would it make sense to organize this better splitting the ones that should be on devDependencies?

As now this is part of a mono-repo, should we remove those that are already in the common package.json and have no version differences?